consensus/module.go

```diff
@@ -0,0 +1,56 @@
+package consensus
+
+import (
+	"log"
+	"pocket/shared/config"
+	"pocket/shared/modules"
+
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+const (
+	DefaultLogPrefix string = "NODE" // Just a default that'll be replaced during consensus operations.
+)
+
+var _ modules.ConsensusModule = &consensusModule{}
+
+type consensusModule struct {
+	pocketBus modules.Bus
+}
+
+func Create(_ *config.Config) (modules.ConsensusModule, error) {
+	m := &consensusModule{
+		pocketBus: nil,
+	}
+	return m, nil
+}
+
+func (m *consensusModule) Start() error {
+	// TODO(olshansky): Add a test that pocketBus is set
+	log.Println("Starting consensus module...")
+	return nil
+}
+
+func (m *consensusModule) Stop() error {
+	log.Println("Stopping consensus module...")
+	return nil
+}
+
+func (m *consensusModule) GetBus() modules.Bus {
+	if m.pocketBus == nil {
+		log.Fatalf("PocketBus is not initialized")
+	}
+	return m.pocketBus
+}
+
+func (m *consensusModule) SetBus(pocketBus modules.Bus) {
+	m.pocketBus = pocketBus
+}
+
+func (m *consensusModule) HandleMessage(anyMessage *anypb.Any) {
+	panic("HandleMessage not implemented")
+}
+
+func (m *consensusModule) HandleTransaction(anyMessage *anypb.Any) {
```

Comment:
I don't understand why transaction needs to be different than message? Isn't a transaction just a message?

Reply:
tl;dr Removing HandleTransaction for now.

Good point. I was thinking of a `Message` as being consensus specific and `Transaction` as being more at the utility level. However, these can both be wrapped in an envelope that'll be opened by the consensus module and we can always add it back later if need be.